### ISSUE TITLE
falsy values become undefined

### ISSUE THREAD

**Issue by fritx** [kaelzhang/node-object-access#2]
```js
obj = { status: 0 }
access(obj, 'status') // --> undefined
```

falsy values like `0`, `false`, `''` all become `undefined`.

My expected behavior is only `undefined` would returns `undefined`.

**Comment by fritx** [comment]
```js
// @param {Array|string} keys
function access (obj, keys, default_) {
  var value
  _access(obj, keys, function (parent, current, key, last) {
    if (last) {
      value = current

      // Actually there is no items to the right, however we return true
      return true
    }

    // Stop accessing.
    if (!current) {
      return true
    }
  })

  // return value || default_
  return value !== void 0 ? value : default_ // <-- my solution
}
```

**Comment by kaelzhang** [comment]
Nice catch~

The fix has been landed in `1.2.2`

**Comment by fritx** [comment]
Cool !
